src/Elastic.Apm/Model/Payload/Request.cs

```diff
@@ -14,7 +14,7 @@ public class Request
     public class Socket
     {
         public bool Encrypted { get; set; }
-        public String Remote_address { get; set; }
+        public String RemoteAddress { get; set; }
```

Comment:
I take it this relies on convention to bind to: `remote_address` we might want to annotate this.